gems/aws-sdk-core/lib/aws-sdk-core/stubbing/protocols/query.rb

```diff
@@ -20,18 +20,18 @@ def stub_error(error_code)
         private
 
         def build_body(api, operation, data)
-          xml = []
-          builder = Aws::Xml::DocBuilder.new(target: xml, indent: '  ')
+          builder = Aws::Xml::DocBuilder.new(target: [], indent: '  ')
           builder.node(operation.name + 'Response', xmlns: xmlns(api)) do
-            if rules = operation.output
+            rules = operation.output
```

Comment:
BTW - Rubocop is okay with assignments in conditionals as long as they are wrapped in parenthesis - `if (rules = operation.output)`